backend/main.py

```diff
@@ -32,7 +33,23 @@
 logger = getLogger("Main")
 
 async def chown_plugin_dir(_):
-    code_chown = call(["chown", "-R", "deck:deck", CONFIG["plugin_path"]])
+    USER = None
+    cmd = "who | awk '{print $1}' | sort | head -1"
```

Comment:
Relating to previous issues found with .Popen, this terminal invocation would preferably be a set of python code. If not possible to use python then measures to ensure the prevention of a zombie process from .Popen should be taken.

Reply:
All code now uses call or check_output.